gradle/release.gradle

```diff
@@ -159,21 +157,19 @@ releaseSteps {
              - see 'TROUBLESHOOTING' section at the top of this file
     */
 
-    step("start operating on $System.env.TRAVIS_BRANCH") { run "git", "checkout",  System.env.TRAVIS_BRANCH }
-
     step("update release notes") { project.notes.updateReleaseNotes() }
 
     step("commit release notes") { commitReleaseNotes(buildInfo) }
-            .rollback { run "git", "reset", "--hard", "HEAD^" }
+            .rollback { run "git", "reset", "--hard", "HEAD~1" }
```

Comment:
imho I prefer by little bit without preference either `HEAD^` or `HEAD~1` which I find easy to understand over `HEAD~`